sdk/storage/Azure.Storage.DataMovement.Blobs/samples/Sample01b_HelloWorldAsync.cs

```diff
@@ -60,19 +61,22 @@ public async Task UploadSingle_ConnectionStringAsync()
                 // Upload file data
                 TransferManager transferManager = new TransferManager(new TransferManagerOptions());
 
+                #region Snippet:UploadSingle_ConnectionStringAsync
                 // Create simple transfer single blob upload job
                 DataTransfer dataTransfer = await transferManager.StartTransferAsync(
                     sourceResource: new LocalFileStorageResource(sourceLocalPath),
                     destinationResource: new BlockBlobStorageResource(destinationBlob));
+                await dataTransfer.AwaitCompletion();
             }
+            #endregion
```

Comment:
Move the `#endregion` to above the bracket. Right now when the README generates it also includes this end bracket.

![image](https://user-images.githubusercontent.com/48961492/206562879-ef59ef53-161c-49b2-9cfd-ded39b5fe3bf.png)

Reply:
Same for the rest of the samples where the `#endregion` comes after the last bracket.